addons/web/static/src/oxp/todo_list.js

```diff
@@ -0,0 +1,56 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+import { Component, useState } from "@odoo/owl";
+
+export class TodoList extends Component {
+    static template = "TodoList";
```

Comment:
maybe change template name to `web.TodoList`, hoping to show them good practices